### PR TITLE
Make brush size independent of zoom value

### PR DESCRIPTION
**Pull request by philippotto**
### URL of deployed dev instance (used for testing):
- https://zoomindependentbrush.webknossos.xyz

### Steps to test:
- set brush size
- zoom in / out
- brush size in nm shouldn't change (focus a vesicle to test, for example)

### Issues:
- fixes https://discuss.webknossos.org/t/brush-tool-size-changes-while-browser-zooming/1346/4

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- [X] Ready for review

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 
 ### Changed
 - Tweaked the highlighting of the active node. The inner node looks exactly as a non-active node and is not round, anymore. An active node is circled by a "halo". In arbitrary mode, the halo is hidden and the active node is round. [#3868](https://github.com/scalableminds/webknossos/pull/3868)
+- Brush size is independent of zoom value, now. This change simplifies volume annotations, as brush sizes can be adapted to certain structures (e.g., vesicles) and don't need to be changed when zooming.
```
Comment (daniel-wer) [inline CHANGELOG.md]: Can you add the link to the PR? :)

File: frontend/javascripts/oxalis/model/volumetracing/volumelayer.js [hunk 2]
```diff
@@ -258,8 +256,7 @@ class VolumeLayer {
     const [scaleX, scaleY] = this.get2DCoordinate(
       getBaseVoxelFactors(state.dataset.dataSource.scale),
     );
-    const zoomFactor = getZoomValue(state.flycam);
-    const scaledBrushSize = state.userConfiguration.brushSize * zoomFactor;
+    const scaledBrushSize = state.userConfiguration.brushSize;
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/model/volumetracing/volumelayer.js]: see last comment, I'd rename to `brushSize`

File: frontend/javascripts/oxalis/model/volumetracing/volumelayer.js [hunk 3]
```diff
@@ -212,9 +211,8 @@ class VolumeLayer {
 
   getCircleVoxelIterator(position: Vector3, boundings?: ?BoundingBoxType): VoxelIterator {
     const state = Store.getState();
-    const zoomFactor = getZoomValue(state.flycam);
     const { brushSize } = state.userConfiguration;
-    const scaledBrushSize = brushSize * zoomFactor;
+    const scaledBrushSize = brushSize;
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/model/volumetracing/volumelayer.js]: `scaledBrushSize` is a bit misleading now, I think this line can simply be removed

**Comment by philippotto** [comment]
Thanks for your feedback!

**Pull request merged**